docs/csharp/index.yml

```diff
@@ -14,57 +14,113 @@ landingContent:
-            url: linq/index.md
-          - text: async and await
-            url: programming-guide/concepts/async/index.md
+          - text: "Type system"
+            url: ./fundamentals/types/index.md
+          - text: "Object oriented programming"
+            url: ./fundamentals/object-oriented/index.md
+          - text: "Functional techniques"
+            url: ./fundamentals/functional/pattern-matching.md
+          - text: "Exceptions"
+            url: ./fundamentals/exceptions/index.md
+          - text: "Coding style"
+            url: ./fundamentals/coding-style/identifier-names.md
       - linkListType: tutorial
         links:
+          - text: Display command-line
+            url: ./fundamentals/tutorials/how-to-display-command-line-arguments.md
+          - text: Intro to classes
+            url: ./fundamentals/tutorials/classes.md
+          - text: Object oriented C#
+            url: ./fundamentals/tutorials/oop.md
           - text: Object-oriented inheritance
             url: ./fundamentals/object-oriented/inheritance.md
+          - text: Converting types
+            url: ./fundamentals/tutorials/safely-cast-using-pattern-matching-is-and-as-operators.md
+          - text: Pattern matching
+            url: ./fundamentals/tutorials/pattern-matching.md
+          - text: Exception handling
+            url: ./fundamentals/exceptions/index.md
           - text: Use LINQ to query data
             url: tutorials/working-with-linq.md
-          - text: Format text using string interpolation
-            url: tutorials/string-interpolation.md
-          - text: Browse all tutorials
-            url: ./fundamentals/tutorials/classes.md
-  
-  - title: "New features in C# 10"
+  - title: "Key Concepts"
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: "Programming Concepts"
+            url: ./programming-guide/concepts/index.md
+      - linkListType: quickstart
+        links:
+          - text: "Methods"
+            url: ./methods.md
+          - text: "Properties"
+            url: ./properties.md
+          - text: "Indexers"
+            url: ./indexers.md
+          - text: "Iterators"
+            url: ./iterators.md
+          - text: "Delegates"
+            url: ./delegates-overview.md
+          - text: "Events"
+            url: ./events-overview.md
+      - linkListType: concept
+        links:
+          - text: "Nullable reference types"
+            url: ./nullable-references.md
+          - text: "Nullable reference migrations"
+            url: ./nullable-migration-strategies.md
+          - text: "Resolve nullable warnings"
+            url: ./nullable-warnings.md
+          - text: "Language Integrated Query (LINQ)"
+            url: linq/index.md
+          - text: "Expression trees"
+            url: ./expression-trees.md
+          - text: "Native interoperability"
+            url: ./programming-guide/interop/index.md
```

Comment:
I'd remove both of these. They are both advanced topics, and neither area gets a lot of traffic.

```suggestion
```